src/application/snapshot/hooks/use-twitter-visible-snapshots.ts

```diff
@@ -10,10 +11,11 @@ interface Properties {
 
 export const useTwitterVisibleSnapshots = ({ hidden }: Properties) => {
   const { tweetAuthors } = useTwitterScraping();
+  const { data: daoHandles } = useDaoHandles('snapshot');
 
   const snapshotUserNodes = useMemo(() => {
-    return getSnapshotUsernameNodes(tweetAuthors);
-  }, [tweetAuthors]);
+    return getSnapshotUsernameNodes(daoHandles ?? {}, tweetAuthors);
+  }, [daoHandles, tweetAuthors]);
```

Comment:
I am thinking that this function `getSnapshotUsernameNodes` is unnecessary now, we need to pass tweetAuthors and daoHandles just to iterate inside and apply another fn. I also have similar feeling to `getSnapshotFromTwitterUsername`, it made more sense when we had constant map but now we are getting that data from api call so maybe we can simplify and do it directly here, something like:
```tsx
  const snapshotUserNodes = useMemo(() => {
    return tweetAuthors.map((author)=>{
      const snapshotName = daoHandles[handle.toLowerCase()];
      return snapshotName ? { snapshotName, ...result} : undefined;
    }).filter(Boolean)
  }, [daoHandles, tweetAuthors]);
```

and you could remove 2 functions mentioned above